pkg/apis/options/providers.go

```diff
@@ -84,6 +84,9 @@ type Provider struct {
 	// The code challenge method
 	CodeChallengeMethod string `json:"code_challenge_method,omitempty"`
 
+	// Allows additional claims to be obtained from the `id_token`.
+	AllowAdditionalClaims []string `json:"allowAdditionalClaims,omitempty"`
```

Comment:
I don't think this a good naming scheme.

In my opinion AllowXX indicates a boolean value instead we should call it AllowedAddiotionalClaims or even better just AdditionalClaims

Reply:
> I don't think this a good naming scheme.
> 
> In my opinion AllowXX indicates a boolean value instead we should call it AllowedAddiotionalClaims or even better just AdditionalClaims

I agree, I have made the correction.